testData/solutions/win32.json5

```diff
@@ -1,4 +1,6243 @@
+                        "schema": {
+                            "title": "Focus locator type",
+                            "description": "Specifies the Focus locator type to use.",
+                            "enum": [
+                                1,
+                                3,
+                                2
+                            ],
+                            "enumLabels": [
+                                "Block",
+                                "Underline",
+                                "Frame"
+                            ],
+                            "default": 2
+                        }
+                    },
+                    "FocusScheme.FocusENHColor": {
+                        "schema": {
+                            "title": "Focus locator color",
+                            "description": "Specifies the locator color.",
+                            "enum": [
+                                -2147483648,
+                                0,
+                                255,
+                                65535,
+                                65280,
+                                16776960,
+                                16711680,
+                                16711935,
+                                16777215
+                            ],
+                            "enumLabels": [
+                                "Invert",
+                                "Black",
+                                "Red",
+                                "Yellow",
+                                "Green",
+                                "Cyan",
+                                "Blue",
+                                "Magenta",
+                                "White"
+                            ],
+                            "default": 255
+                        }
+                    },
+                    "FocusScheme.FocusENHThickness": {
+                        "schema": {
+                            "title": "Focus locator thickness",
+                            "description": "Specifies the thickness of the focus locator.",
+                            "enum": [
+                                1,
+                                5,
+                                9
+                            ],
+                            "enumLabels": [
+                                "Thin",
+                                "Medium",
+                                "Thick"
+                            ],
+                            "default": 5
+                        }
+                    },
+                    "FocusScheme.FocusENHTransparency": {
+                        "schema": {
+                            "title": "Focus locator transparency",
+                            "description": "Specifies the transparency of the focus locator. Range goes from 'Clear' to 'Solid'.",
+                            "type": "integer",
+                            "minimum": 31,
```

Comment:
It seems really suspicious that 31 would be "clear" instead of 0, but fine?

Reply:
Any confirmation here?  This still seems a bit suspicious to me.

Reply:
Ok, the minimum and maximum values you can set in the slider are indeed 31 and 255

<img width="867" alt="image" src="https://user-images.githubusercontent.com/361700/82691397-b0eeb600-9c12-11ea-8612-75f026d8a74f.png">